pom.xml

```diff
@@ -51,7 +51,7 @@
     <formatter.basedir>${project.basedir}</formatter.basedir>
     <classgraph.version>4.8.147</classgraph.version>
     <jsr305.version>3.0.2</jsr305.version>
-    <swagger.version>2.1.13</swagger.version>
+    <swagger.version>2.2.1</swagger.version>
     <swagger.codegen.version>3.0.34</swagger.codegen.version>
     <swagger.codegen.generators.version>1.0.34</swagger.codegen.generators.version>
```

Comment:
This patch breaks the application as some method is missing. I guess we have to bump all Swagger-related versions to make it work.